py3status/core.py

```diff
@@ -356,7 +356,7 @@ def notify_user(self, msg, level='error', rate_limit=None, key=''):
                 pass
         # We use a hash to see if the message is being repeated.  This is crude
```

Comment:
Any issues with this hasher can actually be avoided by taking the hash of a tuple of these three items rather than making them into a string.